### ISSUE TITLE
Querying fails for some sets

### ISSUE THREAD

**Issue by tommiz1989** [aerospike/aerospike-data-browser#5]
We've recently installed the ADB which looks awesome.
On the UI, while running the default query against some sets works as intended, a few sets returns errors.
Also we've seen that these erroneous sets are not showing the data schema.

Testing the data on the aerospike machine using local aql `show set` indeed show that the sets are populated.
![image](https://user-images.githubusercontent.com/19637279/140288873-b19c127d-2dc0-448d-96f4-d5d8756f1fe4.png)

**Comment by reugn** [comment]
@tommiz1989 do you have capital letters in those sets? Please see #4.

**Comment by tommiz1989** [comment]
@reugn Will there be any fix for this issue in the near future?
I saw the ticket you linked is from nearly 3 years back.

**Comment by reugn** [comment]
Yes, we've resolved this issue locally and this will be rolled out to the Data Browser in the near future.

**Comment by ivalexa** [comment]
@reugn Do you have some ETA when it's going to be merged?
